tabs/ansible/grafana/security_group/azure.yaml

```diff
@@ -9,17 +9,17 @@ inputs:
 
   rg_id:
     description: Resource group ID
-    display_label: resource group id
+    display_label: Resource group ID
     type: string
 
   vpc_id:
-    description: VPC id
-    display_label: resource group id
+    description: VPC ID
+    display_label: Virtual network ID
```

Comment:
Not used, why is it here?

Reply:
I resolved it by passing deployment id to security group blueprints and fetch needed parameters from deployment id of parent.